modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js

```diff
@@ -372,98 +364,265 @@ class RuleList extends Component {
-					inputs: Config.object(),
-					label: Config.string(),
-					outputs: Config.object(),
-					target: Config.string(),
-				})
-			),
-			conditions: Config.arrayOf(
-				Config.shapeOf({
-					operands: Config.arrayOf(
-						Config.shapeOf({
-							label: Config.string(),
-							repeatable: Config.bool(),
-							type: Config.string(),
-							value: Config.string(),
-						})
-					),
-					operator: Config.string(),
-				})
-			),
-			['logical-operator']: Config.string(),
-		})
-	).value([]),
-
-	/**
-	 * @default undefined
-	 * @instance
-	 * @memberof RuleList
-	 * @type {!string}
-	 */
-
-	spritemap: Config.string().required(),
-
-	/**
-	 * @default strings
-	 * @instance
-	 * @memberof RuleList
-	 * @type {!object}
-	 */
-
-	strings: Config.object().value({
-		'belongs-to': Liferay.Language.get('belongs-to'),
-		'calculate-field': Liferay.Language.get('calculate-field-x-as-x'),
-		contains: Liferay.Language.get('contains'),
-		'due-to-missing-fields': Liferay.Language.get('due-to-missing-fields'),
-		'equals-to': Liferay.Language.get('is-equal-to'),
-		'greater-than': Liferay.Language.get('is-greater-than'),
-		'greater-than-equals': Liferay.Language.get(
-			'is-greater-than-or-equal-to'
-		),
-		'is-empty': Liferay.Language.get('is-empty'),
-		'less-than': Liferay.Language.get('is-less-than'),
-		'less-than-equals': Liferay.Language.get('is-less-than-or-equal-to'),
-		'not-contains': Liferay.Language.get('does-not-contain'),
-		'not-equals-to': Liferay.Language.get('is-not-equal-to'),
-		'not-is-empty': Liferay.Language.get('is-not-empty'),
-	}),
-};
-
-Soy.register(RuleList, templates);
-
-export default RuleList;
+/**
+ * Prints the DDM form card rule.
+ */
+const EmptyList = ({message}) => {
+    return (
+        <div className="main-content-body">
+            <div className="sheet taglib-empty-result-message">
+                <div className="taglib-empty-result-message-header"></div>
+
+                {message && (
+                    <div className="sheet-text text-center text-muted">
+                        <p className="text-default">{message}</p>
+                    </div>
+                )}
+            </div>
+        </div>
+    )
+}
+
+/**
+ * Prints the DDM form card rule.
+ */
+const Label = ({content}) => {
+    return (
+        <span className="label label-lg label-secondary" data-original-title={content} title={content}>
+            <span className="text-truncate-inline">
+                <span className="text-truncate">{content}</span>
+            </span>
+	    </span>
+    )
+}
+
+
+/**
+ * Prints Rules Conditions.
+ */
+const Condition = ({
+    operandType,
+    operandLabel,
+    operandValue
+}) => {
+    const SubCondition = ({operandType}) => {
+        if(['double', 'integer', 'json', 'option', 'string', 'text', 'numeric'].includes(operandType)){
+            return  (<span>{Liferay.Language.get('value')} </span>);
+        }
+    
+        if(operandType === 'user'){
```

Comment:
What do you think about

```js
const PRIMITIVE_OPERAND_TYPES = new Set(['double', 'integer', 'json', 'option', 'string', 'text', 'numeric']);

const OPERAND_TYPES = new Set(['user','field, 'list']);

if (OPERAND_TYPES.has(operandType)) {
    return (<span>{Liferay.Language.get(operandType)}</span>);
} else if (PRIMITIVE_OPERAND_TYPES.has(operandType)){
    return (<span>{Liferay.Language.get('value')}</span>)
} else {
    return null;
}
```

Reply:
Hi, @diegonvs I made this 
```return (<span>{Liferay.Language.get(operandType)}</span>);``` because portal translation interpolation process.